src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -52,6 +52,8 @@
                 <table class="table table-condensed table-hover">
                     <thead>
                     <tr>
+                      <th>{% trans "CheckIn?" %}
```

Comment:
I think we consistently write it as ``Check-In``, although I'd just drop this label and just leave the checkboxes unlabeled for two reasons: (a) the column can be narrower (b) we can later add multiple actions that operate on the selected rows by just adding multiple buttons, if we want